### PR TITLE
files changed, names adjusted

### PR DESCRIPTION
**Pull request by LippkauFelix**
changed some header names and adjusted the file names. 
Renewable Potentials adjusted.

### CODE REVIEW

File: docs/exogenous/extraction-and-mining.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Extraction and Mining of Fossil Fuels
```
Comment (olejandro) [inline docs/exogenous/extraction-and-mining.md:1]: Sphinx seems to be handling this, but the first header should start with # (i.e. header 1), followed by ## (headear 2), followed by ### (header 3)

File: docs/index.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@ Welcome to ETSAP-TIAM's documentation project!
 ======================================
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
```
Comment (olejandro) [inline docs/index.rst:10]: Is this change because of the # -> ## changes?

File: docs/regional_coverage/demand-projection.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Demand Projection
```
Comment (olejandro) [inline docs/regional_coverage/demand-projection.md:1]: Make it headear 1?

File: docs/restrictions/trading-restrictions.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
-## Trading Restrictions
+## Global Trade of Commodities
```
Comment (olejandro) [inline docs/restrictions/trading-restrictions.md:1]: Should change this header level to level 1 (i.e. only one #)

File: docs/sectoral_coverage/energy-service-demands.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-## Energy-service demands 
+## Energy Service Demands
```
Comment (olejandro) [inline docs/sectoral_coverage/energy-service-demands.md:1]: Also here ## -> #

File: docs/sectoral_coverage/residential.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+## Residential
```
Comment (olejandro) [inline docs/sectoral_coverage/residential.md:1]: ## -> #

**Review by olejandro — COMMENTED** [review]
@LippkauFelix looks like there is some inconsistency in header levels across the files. It should be # for Title/Header 1, ## for Header 2, etc. Could you take a look and adjust?

**Review by LippkauFelix — COMMENTED** [review]
I have changed the header from lvl 2 to lvl 1.

**Pull request merged**